Dockerfile

```diff
@@ -76,4 +76,4 @@ ENV BLOCKY_CONFIG_FILE=/app/config.yml
 
 ENTRYPOINT ["/app/blocky"]
 
-HEALTHCHECK --interval=1m --timeout=3s CMD ["/app/blocky", "healthcheck"]
+HEALTHCHECK --start-period=1m --timeout=3s CMD ["/app/blocky", "healthcheck"]
```

Comment:
It was reduced here from 1m to 30s (default). 10s could be better for most setups, but it will increase load - we start new blocky process on each healthcheck run.

Since HEALTHCHECK is overridable from docker run / compose, I would leave it (use default)